3 reports airline.sql

```diff
@@ -0,0 +1,29 @@
+--1)
+select CountOfPeople = count(*), f.FlightNumber, f.DepartureAirport, f.TimeDeparting, f.ArrivalAirport
+from Flight f
+group by f.FlightNumber, f.DepartureAirport, f.TimeDeparting, f.ArrivalAirport
+
+--2)
+select datediff(day,(getdate()),(TimeDeparting)), *
+from Flight f 
+where f.PassportNumber is null and datediff(day,(getdate()),(TimeDeparting)) between 0 and 7
+
+--3)
+select AmountDeparting = count(distinct f.FlightNumber), DepartureDate = datefromparts(year(TimeDeparting), month(TimeDeparting), day(TimeDeparting)), AmountOfPassengers = count(f.FlightNumber)

```

Comment:
It isn't necessary to do a datefromparts here, just use the timedeparting column alone.